From 6e7d33d02124e52978748ddd9627fdea21ac7d92 Mon Sep 17 00:00:00 2001
From: Matthijs van Duin <matthijsvanduin@gmail.com>
Date: Fri, 9 Sep 2016 18:51:21 +0200
Subject: [PATCH] ARM: am335x: don't reset or idle the rtc on init

This fixes a bus error in omap_hwmod_read called from
omap_rtc_wait_not_busy.  The root cause probably lies elsewhere however
and should be investigated.

Signed-off-by: Matthijs van Duin <matthijsvanduin@gmail.com>
---
 arch/arm/boot/dts/am33xx.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index 2bbfa1fe9e93..5b31a06b2fe8 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -583,6 +583,8 @@
 			interrupts = <75
 				      76>;
 			ti,hwmods = "rtc";
+			ti,no-reset-on-init;
+			ti,no-idle-on-init;
 		};
 
 		spi0: spi@48030000 {
-- 
2.9.3

